src/applications/debt-letters/sass/debt-letters.scss

```
@@ -33,5 +33,10 @@ a i {
   &:visited {
     color: #0071bc;
```

This should be purple

```suggestion
    color: $color-visited;
```